hub/powertoys/advanced-paste.md

```diff
@@ -118,24 +118,15 @@ If pasting files is accepted within the application that you are using (e.g. Fil
 
 ### Paste as .html file
 
-**Paste as .html file** enables you to paste text stored in your clipboard as a .html file with an auto-generated file name. You can optionally set a quick key shortcut in settings.
+**Paste as .html file** enables you to paste html data stored in your clipboard as a .html file with an auto-generated file name. This is especially useful for saving a part of a web-page from a browser - including links, formatted text and images. You can optionally set a quick key shortcut in settings.
 
 Sample input:
 
-```html
-<!DOCTYPE html>
-<html>
-<head>
-    <title>PowerToys</title>
-</head>
-<body>
-    <h1>PowerToys</h1>
-    <p>PowerToys is a set of utilities for power users to tune and streamline their Windows experience for greater productivity.</p>
-</body>
-</html>
-```
+<pre>
```

Comment:
Why remove the markdown syntax block? We should only use HTML in the .md pages when there's no markdown equivalent.

Reply:
The markdown syntax block is inappropriate for Paste as .html. When you hit the "copy" button for the generated code-block, it copies *text* to the clipboard. But Paste as .html needs HTML data, not text (even text that corresponds to HTML code won't do the trick unfortunately). If you feel strongly against the `<pre>` block, then we should remove the sample here entirely.

Reply:
That's a good point. I don't think it makes sense to have a Sample Input block for this section. Best to remove it. The info you added to the description gives the use an idea of how it can be used.

Reply:
Done